client/src/components/graph/graph.js

```diff
@@ -757,7 +767,35 @@ class Graph extends React.PureComponent {
             right: globals.leftSidebarWidth
           }}
         >
-          <div id="graphAttachPoint" />
+          <div id="graphAttachPoint">
+            <GraphOverlayLayer
+              cameraTF={cameraTF}
+              modelTF={modelTF}
+              projectionTF={projectionTF}
+              graphPaddingRightLeft={this.graphPaddingRightLeft}
+              graphPaddingTop={this.graphPaddingTop}
+              responsive={responsive}
+            >
+              <CentroidLabels
+                labels={centroidLabels.labels}
+                dilatedValue={pointDilation.categoryField}
+                colorAccessor={colorAccessor}
+                dispatch={dispatch}
```

Comment:
why pass dispatch explicitly, and not just use the @connect decorator when defining the CentroidLables component?

Reply:
I think I misinterpreted what a pure component was.  I connected the component to the redux store, which also allowed me to remove all the props given inside of the graph component.